node/tests/RedisClusterClient.test.ts

```diff
@@ -321,4 +321,45 @@ describe("RedisClusterClient", () => {
             client.close();
         },
     );
+
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "object freq transaction test_%p",
+        async (protocol) => {
+            const client = await RedisClusterClient.createClient(
+                getClientConfigurationOption(cluster.getAddresses(), protocol),
+            );
+
+            const key = uuidv4();
+            const maxmemory_policy_key = "maxmemory-policy";
```

Comment:
I think typescript usually uses camel-case, should we change the variable names in this test to align with that?
```suggestion
            const maxmemoryPolicyKey = "maxmemory-policy";
```